api/pkg/s3/datastore/aws/aws.go

```diff
@@ -165,6 +165,26 @@ func (ad *AwsAdapter) DELETE(object *pb.DeleteObjectInput, ctx context.Context)
 	return NoError
 }
 
+func (ad *AwsAdapter) RestoreObject(object *pb.RestoreObjectInput, ctx context.Context) S3Error {
+
+	bucket := ad.backend.BucketName
+
+	newObjectKey := object.Bucket + "/" + object.Key
+
+	restoreInput := awss3.RestoreObjectInput{Bucket: &bucket, Key: &newObjectKey}
+
+	svc := awss3.New(ad.session)
+	_, err := svc.RestoreObject(&restoreInput)
```

Comment:
why are we ignoring the important information in the first response parameter ?
`type RestoreObjectOutput struct {

    // If present, indicates that the requester was successfully charged for the
    // request.
    RequestCharged *string `location:"header" locationName:"x-amz-request-charged" type:"string" enum:"RequestCharged"`

    // Indicates the path in the provided S3 output location where Select results
    // will be restored to.
    RestoreOutputPath *string `location:"header" locationName:"x-amz-restore-output-path" type:"string"`
    // contains filtered or unexported fields
}`